Port 5000 already in use by AirPlay on macOS 12

**Description**
Port 5000 used as the server port in the project is already in use by AirPlay on macOS 12. As such, when I run `npm run dev`, this the following error message is shown:
[ERROR] 14:44:54 Error: listen EADDRINUSE: address already in use :::5000
![image](https://user-images.githubusercontent.com/67887489/146724275-baf11501-da4c-458a-84c2-b02a7ab515a4.png)

**Additional context**
Brief Googling reveals that this is an issue with Apple using port 5000 for AirPlay Receiver. See [here](https://developer.apple.com/forums/thread/682332) and [here](https://www.reddit.com/r/webdev/comments/qg8yt9/apple_took_over_port_5000_in_the_latest_macos/). 

I don't fully understand this, but apparently this is [not an unreasonable decision](https://www.reddit.com/r/webdev/comments/qg8yt9/comment/hi65w7i/):
> Just did a bit more digging, and IANA actually assigns port 5000 to "commplex-main" - so really any other service that's using that port is violating the official assignment. Based on a quick search of what commplex-main is, it's related to UPnP for network device discoverability. I'm not sure exactly what the AirPlay Receiver service does, but I get the gist that it probably does fit somewhere into the scope of UPnP-related features. At the very least, it's probably a far more accurate usage of port 5000 than some random dev servers.

**Solution**
A quick way to fix the problem is to disable AirPlay Receiver in System Settings > Sharing. 

However, I was thinking a better solution might be to simply use a different port number for the server. If I'm not mistaken, we can just use another port number that is less likely to result in such conflicts? If so, happy to submit a PR to change all instances of 5000 to, say, another randomly chosen port number.

--- a/.env.example
+++ b/.env.example
@@ -2,7 +2,7 @@ DB_HOST=localhost
 DB_NAME=askgov
 DB_USER=root
 DB_PASSWORD=password
-SERVER_PORT=5000
+SERVER_PORT=6174
 NODE_ENV=development
 SESSION_SECRET=mySessionSecret
 

--- a/README.md
+++ b/README.md
@@ -106,7 +106,7 @@ Optionally [DBeaver](https://dbeaver.io/download/) to view database with GUI
   
   Frontend server accessible on `localhost:3000`
   
-  Backend server accessible on `localhost:5000/api/v1`
+  Backend server accessible on `localhost:6174/api/v1`
   
   Local mail server `MailDev` accessible on `localhost:1080`
 
@@ -158,7 +158,7 @@ Optionally [DBeaver](https://dbeaver.io/download/) to view database with GUI
 
 ## API Endpoints
 
-#### Base Url - `http://localhost:5000/api/v1`
+#### Base Url - `http://localhost:6174/api/v1`
 
 #### Agencies
 - `GET /agencies?<longname, shortname>`

--- a/client/package.json
+++ b/client/package.json
@@ -57,7 +57,7 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:5000",
+  "proxy": "http://localhost:6174",
   "devDependencies": {
     "@types/draftjs-to-html": "^0.8.0",
     "@types/html-to-draftjs": "^1.4.0",

--- a/server/src/bootstrap/config/base.ts
+++ b/server/src/bootstrap/config/base.ts
@@ -23,7 +23,7 @@ const baseSchema: Schema<BaseConfig> = {
   serverPort: {
     doc: 'Application port',
     format: 'port',
-    default: 5000,
+    default: 6174,
     env: 'SERVER_PORT',
   },
   logoHost: {

--- a/server/src/bootstrap/index.ts
+++ b/server/src/bootstrap/index.ts
@@ -77,7 +77,7 @@ app.use(requestLoggingMiddleware)
 
 // cors enable
 app.options('*', cors<express.Request>())
-app.use(cors({ origin: 'http://localhost:5000' }))
+app.use(cors({ origin: `http://localhost:${process.env.SERVER_PORT}` }))
 
 // security config
 app.use(helmet(helmetOptions))

